.github/workflows/molecule.yml

```diff
@@ -90,10 +88,11 @@ jobs:
       - name: molecule
         uses: robertdebock/molecule-action@6.0.1
         env:
-          MOLECULE_DISTRO: "${{ matrix.distros }}"
           TEST_RELEASE: "${{ matrix.releases }}"
           TEST_VERSION: "${{ matrix.versions }}"
+          MOLECULE_PLATFORM_NAME: rclone-archlinux
         with:
+          image: carlodepieri/docker-archlinux-ansible
```

Comment:
We should be able to remove this since it is configured in platform.

Reply:
correct, that was just some random try yesterday

Reply:
while we're at it: I think `galaxy_release.yml` in the workflows could be removed also. Was a try back then, the task is done within `molecule.yml`. Agree?